health_monitoring/test_health_monitor.py

```diff
@@ -404,3 +404,176 @@ def test(self):
         s = self.get_server("deproxy")
         s.drop_conn_when_receiving_data = True
         time.sleep(1)
+
+
+class TestHmCrcAutoImplicit(tester.TempestaTest):
```

Comment:
You should [parametrize](https://github.com/tempesta-tech/tempesta-test/blob/c1bb410e1dba70c9ba0ae86eb0c9b05e48f6483e/t_regress/test_reboot_under_load.py#L55) here. You can use tempesta config, delay as params.